console/console-init/ui/src/modules/iot-device/components/CreateMetadata/MetaDataRow.tsx

```diff
@@ -29,16 +29,43 @@ export const MetaDataRow: React.FC<IMetaDataRow> = ({
   rowIndex
 }) => {
   const metadataRow = metadataList[rowIndex];
-  // console.log("metadataList", metadataList, "type", metadataRow.type);
+  const [validationStatus, setValidationStatus] = useState<"default" | "error">(
```

Comment:
I think you can use ValidatedOptions for default/error/sucess. 
import {ValidatedOptions} from "@patternfly/react-core";

Reply:
String values are accepted values in 'validated' props.